Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -722,3 +722,26 @@
     attributes:
       proper: true
       gender: male
+
+- type: entity
+  name: Squirt
+  parent: MobTurtle
+  id: MobTurtleSquirt
+  description: She's a long way from the ocean...
+  components:
+  - type: InteractionPopup
+    successChance: 0.6
+    interactSuccessString: petting-success-kangaroo
```

Comment:
kangaroo?

Reply:
ehh, we've already got Paperwork whos a slow station pet. Also we have the water entity, so we could use at least one mob for that, and I think this is at least better than a fish. (right now all we have is crab.)